resources/views/admin.blade.php

```diff
@@ -134,13 +162,25 @@ function deleteEmployee(id){
                 }
 
                 //Create Employee Axios
-                $('#modal-form').submit(function (e) {
+                $('#btn-add-employee').click(function (e) {
+                    axios.get('employee/create')
+                    .then(function (response){
+                        const companies = response.data
+                        const select = $('#select-create')
+                        companies.forEach(company => {
```

Comment:
pass the companies data via controller to blade and set the options directly in select element on the blade without appending @axnish27.

Reply:
Yh sure @Raza9798

Reply:
Requested changes have been made please review again @Raza9798